mlflow/xgboost/__init__.py

```diff
@@ -376,10 +377,10 @@ def autolog(
     :param silent: If ``True``, suppress all event logs and warnings from MLflow during XGBoost
                    autologging. If ``False``, show all events and warnings during XGBoost
                    autologging.
+    :param max_tuning_runs: Integer or None, passed to ``mlflow.sklearn`` autologging routine,
+                            if the model to be logged is XGBoost scikit-learn estimators.
```

Comment:
I don't think `max_tuning_runs` is relevant for XGBoost scikit-learn estimators, as it's only used for `GridSearchCV` and `RandomizedSearchCV` from scikit-learn. This parameter & docstring can be removed.

Reply:
Yes. I think you are right. Thanks for catching it!